.internal-ci/util/manual_republish_existing_build.sh

```diff
@@ -6,90 +6,210 @@
 #   Use with caution.
 
 set -e
-set -x
 
-echo "no foot-guns, check the tag vars and remove this exit 0 to use!"
-exit 0
+usage()
+{
+cat << USAGE
+Usage:
+${0} [--pull] [--images] [--charts] --source-tag v1.2.2-dev --target-tag v1.2.3-some-tag
```

Comment:
Add `push`, `image` and `chart`, too
```suggestion
${0} [--pull] [--push] [--images|--image IMG] [--charts|--chart CHART] --source-tag v1.2.2-dev --target-tag v1.2.3-some-tag
```